CHANGELOG.md

```diff
@@ -34,6 +34,19 @@ changes.
   + Will make bumping dependencies (e.g. cardano-node) easier.
   + Build commands change, see updated [Contribution Guidelines](https://github.com/input-output-hk/hydra/blob/master/CONTRIBUTING.md)
 
+- **BREAKING** Change the way tx validity and contestation deadline is constructed for close transactions:
+  + There is a new hydra-node flag `--contestation-period` expressed in seconds
+    to control the close tx validity bounds as well as determine the
+    contestation deadline.
+  + The deadline is always `2 * --contestation-period` after the Close tx.
```

Comment:
this is not true anymore with `min(maxGraceTime, contestationPeriod)`. Instead it may be "up to 2*cp"